src/dev/dev_build_cache.ts

```diff
@@ -174,7 +175,7 @@ export class DiskBuildCache implements DevBuildCache {
 
   addUnprocessedFile(pathname: string): void {
     this.#unprocessedFiles.set(
-      pathname,
+      pathname.replaceAll(WINDOWS_SEPARATOR, "/"),
```

Comment:
I don't know if there is a "better" way to normalize file system paths to URL-like paths. This seems to work at least, since manually replacing `\\` with `/` in my `snapshot.json` locally fixes the issue.

Reply:
Seems fine to me.